website/docs/r/vra_storage_profile_aws.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: "vra"
+page_title: "VMware vRealize Automation: vra_storage_profile_aws"
+sidebar_current: "docs-vra-resource-vra-storage-profile_aws"
+description: |-
+  Provides a data lookup for vra_storage_profile_aws.
+---
+
+# Resource: vra_storage_profile_aws
+## Example Usages
+This is an example of how to create a storage profile aws resource.
+
+**Vra storage profile aws:**
+
+```hcl
+# AWS storage profile using generic vra_storage_profile resource. Use 'vra_storage_profile_aws' resource as an alternative.
+resource "vra_storage_profile" "this" {
```

Comment:
The example is for creating AWS storage profile using generic `vra_storage_profile`resource. This example along with Azure one using generic `vra_storage_profile` resource should be included in the storage profile resource page.

Replace this example to use `vra_storage_profile_aws` resource and include for both instance store and EBS.